components/brave_rewards/resources/ui/components/donate/index.tsx

```diff
@@ -45,13 +45,29 @@ export interface Props {
 
 interface State {
   missingFunds: boolean
+  amountChanged: boolean
 }
 
 export default class Donate extends React.PureComponent<Props, State> {
+  private sendButton: HTMLButtonElement | null
+
   constructor (props: Props) {
     super(props)
     this.state = {
-      missingFunds: false
+      missingFunds: false,
+      amountChanged: false
+    }
+    this.sendButton = null
+  }
+
+  componentDidMount () {
+    if (this.sendButton) {
+      this.sendButton.focus()
+    }
+
+    const selectedAmount: HTMLElement | null = document.querySelector('[data-selected="selected"]')
```

Comment:
I was not able to make a dynamic node reference for the initial selected amount. It needs to validate so the "not enough funds" message shows up if the user has zero bat/less than the initial select.